src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java

```diff
@@ -118,21 +185,19 @@ public String getName() {
     }
 
     private boolean isEnabled(Level level) {
-        return levelEnabledFor(level, outLevel, errLevel, logger);
+        int severity = level.getSeverity();
+        if (severity < minSeverity) return false;
+        return levelEnabledFor(level, config, logger);
     }
 
     @Override
     public final boolean on() {
         return debugOn;
     }
 
-    static boolean levelEnabledFor(Level level, Level outLevel, Level errLevel,
-                                   System.Logger logger) {
-        if (level == Level.OFF) return false;
-        int severity = level.getSeverity();
-        return severity >= errLevel.getSeverity()
-                || severity >= outLevel.getSeverity()
-                || logger.isLoggable(level);
+    static boolean levelEnabledFor(Level level, LoggerConfig config,
```

Comment:
It looks like this can be made `private`

Reply:
Done